packages/webpack-cli/lib/groups/ZeroConfigGroup.js

```diff
@@ -27,13 +34,27 @@ class ZeroConfigGroup extends GroupHelper {
                 );
                 return this.args.mode;
             }
+            if (isNoMode && this.args.mode) {
+                logger.warn(
+                    'You Provided both mode and no-mode arguments. You Should Provide just one. "mode" will be used.'
+                )
+            }
+            if (isNoMode && (this.args.dev || this.args.prod)) {
+                logger.warn(
```

Comment:
There's already the same message in this file. Please reuse it instead of copying